demo/universal/js/ReactServerDOMWebpack.re

```diff
@@ -0,0 +1,64 @@
+/* TODO: Move this bindings into reason-react */
+type callServerCallback('a, 'b) = (string, list('a)) => Js.Promise.t('b);
```

Comment:
could you rename 'a and 'b to be something meaningful? I applies to the entire module, I t takes a bit to follow all the a/b and createServerReferenceImpl generics